File: pdaily-server-postgresql/src/test/java/co/icesi/pdaily/testing/PDailyTest.java
package co.icesi.pdaily.testing;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

import org.junit.jupiter.api.extension.ExtendWith;

import co.haruk.core.testing.HarukTest;

/**
 * @author andres2508 on 8/11/19
 **/
@HarukTest
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@ExtendWith(PDailyTestingExtension.class)
public @interface PDailyTest {
}

File: backend/backend/src/test/kotlin/co/pdaily/cloud/testing/PdailyTest.kt
package co.pdaily.cloud.testing

import org.junit.jupiter.api.extension.ExtendWith
import javax.enterprise.util.Nonbinding

@ExtendWith(PdailyTestExtension::class)
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class PdailyTest(@get:Nonbinding val sqlDataSets: Array<String> = [])
